packages/components/src/internal/components/report-list/__snapshots__/ReportList.spec.tsx.snap

```diff
@@ -4,24 +4,24 @@ exports[`<ReportItemModal /> ReportItemModal renders 1`] = `
 "<ReportItemModal report={{...}} onClose={[Function: noop]}>
   <div className=\\"report-item-modal\\">
     <Modal show={true} onHide={[Function: noop]} backdrop={true} keyboard={true} autoFocus={true} enforceFocus={true} restoreFocus={true} manager={{...}} renderBackdrop={[Function: renderBackdrop]} animation={true} dialogComponentClass={[Function: ModalDialog]} bsClass=\\"modal\\">
-      <Modal onHide={[Function: noop]} keyboard={true} autoFocus={true} enforceFocus={true} restoreFocus={true} manager={{...}} renderBackdrop={[Function: renderBackdrop]} show={true} containerClassName=\\"modal-open\\" transition={[Function: DialogTransition]} backdrop={true} backdropTransition={[Function: BackdropTransition]} backdropClassName=\\"modal-backdrop\\" onEntering={[Function: bound handleEntering]} onExited={[Function: bound handleExited]} onMouseUp={[Function (anonymous)]}>
-        <Portal container={[undefined]} onRendered={[Function (anonymous)]}>
+      <Modal onHide={[Function: noop]} keyboard={true} autoFocus={true} enforceFocus={true} restoreFocus={true} manager={{...}} renderBackdrop={[Function: renderBackdrop]} show={true} containerClassName=\\"modal-open\\" transition={[Function: DialogTransition]} backdrop={true} backdropTransition={[Function: BackdropTransition]} backdropClassName=\\"modal-backdrop\\" onEntering={[Function: bound handleEntering]} onExited={[Function: bound handleExited]} onMouseUp={[Function]}>
+        <Portal container={[undefined]} onRendered={[Function]}>
```

Comment:
I've noticed a few times in recent PRs that these ReportList related snapshots keep getting updated without any related chagnes or package updates. Have you done a `yarn install` lately in your local labkey-ui-components repo? I think you might have some out of date packages in your node_modules folder.

Reply:
I just rebased onto the 104.0 release, npm i'd, ran tests with --u, and reran ReportList.spec.tsx for good measure. Weirdly enough, this didn't give me any new snapshots to update. I agree that these changes look unrelated to updates in this branch.

Reply:
@RosalineP 
npm installed? This package uses yarn, so if you used npm, you might have gotten a package-lock.json file locally which you likely will want to delete. Then try `yarn cleanAll` and then `yarn install`. See if that does it and if these spanshot tests revert locally.

Reply:
Oop, it sure does! Thank you